roles/atomic_scan_verify/tasks/main.yml

```diff
@@ -0,0 +1,43 @@
+---
```

Comment:
If that is the case, does it make sense to run this on anything other than RHELAH?  We can keep it as is and when RHEL containers are no longer available, I can add a when clause to the role.